scripts/context.js

```diff
@@ -54,7 +62,11 @@ const paginationPathWithPrefix = curry((pfx, page, totalPages) => {
  */
 const createPaginatedContext = curry(
   (template, postsPerPage, posts, paginationPath, prefix, tag) => {
-    const numPages = Math.ceil(divide(posts.length, postsPerPage));
+    const { length: postsLength } = posts;
+    if (postsLength <= 0) {
```

Comment:
do it other way around, if larger than 0 then procees, since it's only 1 case, could be more readable with R.when

Reply:
There's also https://ramdajs.com/docs/#gte and similar -- often the param order is opposite what you'd expect, so YMMV